Day-I-Social-Card/social-card/src/components/FooterComponents/FooterContent.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './Footer.css';
+
+function FooterContent(props) {
+  return (
+    
+	<React.Fragment>
+    	<div className="comments">{props.numberOfComments}</div>
```

Comment:
It looks like there's some inconsistent indentation here; I must emphasize the usefulness of [prettier](https://prettier.io)!